learningequality/kolibri-design-system
# Add new icons

## Description



**1. Expose the following Google Material Icons**

![Screenshot from 2021-07-13 15-54-00](https://user-images.githubusercontent.com/13509191/125464241-6a769e3c-3ce1-413a-aed9-da16edad1c6c.png)

![Screenshot from 2021-07-13 15-55-04](https://user-images.githubusercontent.com/13509191/125464417-dcf2322c-f1ad-4913-bdf1-589eeaa7aa3d.png)

![Screenshot from 2021-07-13 15-55-41](https://user-images.githubusercontent.com/13509191/125464482-e1c1f6d7-65cd-4501-b07b-cf9acb944235.png)

![Screenshot from 2021-07-13 15-56-22](https://user-images.githubusercontent.com/13509191/125464583-ccb72445-2960-45ef-9fd3-6284836a0713.png)

**2. Add the a11y custom icon**

![Screenshot from 2021-07-13 15-57-57](https://user-images.githubusercontent.com/13509191/125464828-09c1c5ed-c0e4-48a6-befc-2df6d4e7c952.png)

---

As this PR only adds new icons, no updates in Kolibri, KDP, and Studio are needed.

#### Issue addressed


We need the material icons in Kolibri new features and the a11y icon has been added on @radinamatic's request.

## Steps to test

Check [the icons documentation](https://deploy-preview-252--kolibri-design-system.netlify.app/icons/#icons)

## Reviewer guidance



- [ ] Do icon aliases make sense?
